Added and Updated Network Logos

purevertigo opened the pull request:
Just leant how to contribute!

p0psicles commented:
Because this PR is bigger. Can you describe whats the reasoning behind the changes?
Or at least I noticed some of the logo's are in white now. You tested with dark and light theme?

purevertigo commented:
Hi @p0psicles I have updated several logos to the 2020 versions and added some that were not previously in the database/ Some of the logos in white (official colour) have been tested and confirmed to work properly.

thanks!
